crates/bifrost/src/providers/replicated_loglet/sequencer/appender.rs

```diff
@@ -235,119 +253,224 @@ impl<T: TransportConnect> SequencerAppender<T> {
             .replicated_loglet
             .log_server_rpc_timeout;
 
-        let timeout_at = MillisSinceEpoch::after(store_timeout);
         // track the in flight server ids
         let mut pending_servers = NodeSet::from_iter(spread.iter().copied());
-        let mut store_tasks = FuturesUnordered::new();
```

Comment:
Out of curiosity: Did you observe a problem with `FuturesUnordered` or is this a precaution?